apps/src/code-studio/pd/application/teacher/ChooseYourProgram.jsx

```diff
@@ -47,6 +47,17 @@ const CourseHoursLabeledNumberInput = props => {
   );
 };
 
+const TeachingPlansNote = () => {
+  return (
+    <p style={styles.error}>
+      Note: This program is designed to work best for teachers who are teaching
```

Comment:
Nit: In the spec, this reads as "Scholarship eligibility is *often* dependent" ... not sure if the change was intentional

Reply:
Nice catch, thank you! The eslint auto-fixes was sometimes deleting words so I definitely missed this one!